docs/src/hohqmesh_tutorial.md

```diff
@@ -0,0 +1,492 @@
+
+With this newly generated mesh we are ready to run a Trixi.jl simulation on an unstructured quadrilateral mesh.
+For this we must create a new elixir file.
+
+The elixir file given below creates an initial condition for a
+uniform background flow state with a free stream Mach number of 0.3.
+A focus for this part of the tutorial is to specify the boundary conditions and to construct the new mesh from the
+file that was generated in the previous exercise.
+
+It is straightforward to set the different boundary
+condition types in an elixir by assigning a particular function to a boundary name inside a
+Julia dictionary, `Dict`, variable. Observe that the names of these boundaries match those provided by HOHQMesh
+either by default, e.g. `Bottom`, or user assigned, e.g. `IceCream`. For this problem setup use
+* Freestream boundary conditions on the four box edges.
+* Free slip wall boundary condition on the interior curved boundaries.
+
+To construct the unstructured quadrilateral mesh from the HOHQMesh file we point to the appropriate location
+with the variable `mesh_file` and then feed this into the constructor for the `UnstructuredMesh2D` type in Trixi.jl
+```julia
+# create the unstructured from your mesh file
+mesh_file = joinpath(@__DIR__, "out", "ice_cream_straight_sides.mesh")
+mesh = UnstructuredMesh2D(mesh_file)
+```
+
+The complete elixir file for this simulation example is given below.
+```julia
+using OrdinaryDiffEq
+using Trixi
+
+equations = CompressibleEulerEquations2D(1.4) # set gas gamma = 1.4
+
+# freestream flow state with Ma_inf = 0.3
+@inline function uniform_flow_state(x, t, equations::CompressibleEulerEquations2D)
+
+  # set the freestream flow parameters
+  rho_freestream = 1.0
+  u_freestream = 0.3
+  p_freestream = inv(equations.gamma)
+
+  theta = 0.0 # zero angle of attack
+  si, co = sincos(theta)
+  v1 = u_freestream * co
+  v2 = u_freestream * si
+
+  prim = SVector(rho_freestream, v1, v2, p_freestream)
+  return prim2cons(prim, equations)
+end
+
+# initial condition
+initial_condition = uniform_flow_state
+
+# boundary condition types
+boundary_condition_uniform_flow = BoundaryConditionDirichlet(uniform_flow_state)
+boundary_condition_slip_wall = BoundaryConditionWall(boundary_state_slip_wall)
+
+# boundary condition dictionary
+boundary_conditions = Dict( :Bottom     => boundary_condition_uniform_flow,
+                            :Top        => boundary_condition_uniform_flow,
+                            :Right      => boundary_condition_uniform_flow,
+                            :Left       => boundary_condition_uniform_flow,
+                            :LeftSlant  => boundary_condition_slip_wall,
+                            :RightSlant => boundary_condition_slip_wall,
+                            :IceCream   => boundary_condition_slip_wall )
+
+# DGSEM solver.
+#    1) polydeg must be >= the polynomial order set in the HOHQMesh control file to guarantee
```

Comment:
Isn't it 
```suggestion
#    1) polydeg must be >= two times the polynomial order set in the HOHQMesh control file to guarantee
```
or was that just for certain mesh types (with hanging nodes?)?

Reply:
Sort of, this is related to 3D curvilinear meshes where there are stronger constraints on the mapping. For 2D curvilinear `polydeg >= polydeg_mesh` is sufficient.

Reply:
Thanks for the clarification!